_content/implementations.md

```diff
@@ -34,11 +34,11 @@ These projects are reference implementations maintained by the protocol develope
     </a>
 
     <div class="author">&mdash; by <a href="/support.html">tus</a></div>
-
+    
```

Comment:
In this file, there is a lot of whitespace added. Maybe your editor did that on its own but - if possible - please remove the whitespace again. We have a convention that if a line contains no text, it should also contain no whitespace :)